.bumpversion_contracts.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.20.0
+current_version = 0.21.0
```

Comment:
This is the contracts_version upgrade.  We have a contracts_version in addition to the PyPI package version because the same PyPI package version must provide multiple contracts_version for the client.  This is because the client can use different contracts_versions for different modes (DEVELOPMENT and PRODUCTION)l.